linera-storage-service/src/common.rs

```diff
@@ -28,6 +28,8 @@ pub enum KeyTag {
     Key = MIN_VIEW_TAG,
     /// Prefix for the storage of existence or not of the namespaces.
     Namespace,
+    /// Prefix for the root key
+    RootKey,
```

Comment:
The first variant should probably be called `RootView`.
* "Prefix for the storage of the keys of the map" -> "Key prefix for the storage of the root views"
* etc

Reply:
I am against putting the word "view". It is true that storage can be used for views but that is not the only usage.
I understand that the `MIN_VIEW_TAG` can give the wrong impression and so I removed it.